Fixed bug

DylanBulmer opened the pull request:
I fixed the bug I encountered in isssue #33

Commit:
Fixed bug where every element is removed

DylanBulmer commented:
I only changed the file within the `src` folder, I didn't want to change the items in the `dist` folder.

kamlekar commented:
Hi @DylanBulmer ,

Thanks for the PR. much appreciated!!

Can you please change your PR to not remove variable declaration from everywhere? I don't want to have conflicts with other global variables from somewhere else because of addition of this library.

Commit:
Update slimscroll.js

DylanBulmer commented:
I'm not changing any global variables I thought. I added in a comment in the beginning to help the intelli-sense let me know how to handle different parts of the code. I removed some unnecessary parentheses and the `var` keyword that was in front of already declared variables from the parameter lists. 

I can re-upload the code without those changes if you'd like.

DylanBulmer reviewed (commented):
I only tabbed an un-indented piece of code

kamlekar commented:
@DylanBulmer you aren't changing any global variables but if the user's project has global variable `e`, that will lead to conflicts. It is better to write the library code specifically always. Can you please leave the `var` as it is?

kamlekar commented:
Hi @DylanBulmer just tested the code, the test1 is failing. FYI, there are two test folders which needed to pass for me to accept this PR.

Please check by: 
- Moving the scrollbar (with mousewheel)
- Clicking in the scroll region, to move the scroll bar automatically at that position
- Dragging the scrollbar with mouse
- Resizing the window and checking (for test2)

Please let me know if you need more help.

Thanks.

DylanBulmer commented:
Ah, ok, I can look into that, it might take a little bit since finals week is this week, but I'll make sure that I'll fix it so both tests pass.
